src/main/java/fr/insee/pearljam/batch/dao/CommunicationMetadataDao.java

```diff
@@ -0,0 +1,9 @@
+package fr.insee.pearljam.batch.dao;
+
+import fr.insee.pearljam.batch.campaign.CommunicationMetadataType;
+
+public interface CommunicationMetadataDao {
+
+	void createMetadata(CommunicationMetadataType metadata, String surveyUnitId);
```

Comment:
suggestion: could bulk create all Metadatas for each SurveyUnit => else could lead to multiple I/O with DB